docs/source/futures.rst

```diff
@@ -721,6 +721,7 @@ providing an additional channel of communication between ongoing tasks.
 
 .. autoclass:: Pub
    :members:
+   :noindex:
```

Comment:
Since we already have an `autoclass` entry for `Pub` further down, we can probably just remove the duplicate `autoclass` directive here altogether (this is similar to what we do with other coordination primitives like `Semaphore`)

Reply:
Thanks for the advice, I was wondering the same thing. I removed the `autoclass` directive for this in 0f80e85.